app/addons/dashboard/stores.js

```diff
@@ -0,0 +1,139 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+define([
+  'app',
+  'api',
+  'addons/dashboard/actiontypes',
+  'addons/dashboard/resources'
+], function (app, FauxtonAPI, ActionTypes, Resources) {
+
+  var DashboardStore = FauxtonAPI.Store.extend({
+
+    dashboardWidgetActiveTaskInitialize: function (collectionTable, backboneCollection) {
+      this.reset(collectionTable, backboneCollection);
+    },
+
+    reset: function (collectionTable, backboneCollection) {
+      this._prevSortbyHeader = 'started_on';
+      this._headerIsAscending = true;
+      this._sortByHeader = 'started_on';
+      this._collection = collectionTable;
+      this._pollingIntervalSeconds = 5;
+      this.sortCollectionByColumnHeader(this._sortByHeader);
+      this._backboneCollection = backboneCollection;
+    },
```

Comment:
take  these out of this file, and anywhere they are mentioned in this file, they are not needed 
    this._prevSortbyHeader = 'started_on'; 
    this._headerIsAscending = true;
    this._sortByHeader = 'started_on';
    this.sortCollectionByColumnHeader(this._sortByHeader);